src/Pages/WhyVacatePage.jsx

```diff
@@ -1,26 +1,293 @@
-import { Container } from "@mui/material";
-import HeroPanel from "../Components/HeroPanel";
-import WhyVacateAssistance from "./WhyVacateAssistance";
-import WhyVacateEducation from "./WhyVacateEducation";
-import WhyVacateEmployment from "./WhyVacateEmployment";
-import WhyVacateHousing from "./WhyVacateHousing";
+import { Box, Container, Divider, Grid, Typography } from "@mui/material";
+import RedesignHeroPanel from "../Components/RedesignHeroPanel";
+
+import ResponsiveJumpButtonGroup from "../Components/ResponsiveJumpButtonGroup";
+import FactImageCard from "../Components/FactImageCard";
+
+import { useResourcesStyles } from "../Styles/useResourcesStyles";
+
+import teamBuilding from "./../Assets/team_building.svg";
+import cityBuildings from "./../Assets/city.svg";
+import time from "./../Assets/time.svg";
+import emails from "./../Assets/sending_emails.svg";
+import question from "./../Assets/question.svg";
+import dream from "./../Assets/dream.svg";
+import diversity from "./../Assets/diversity.svg";
+import processBuilding from "./../Assets/process.svg";
+import career from "./../Assets/career.svg";
+import graduation from "./../Assets/graduation.svg";
+import FactSubList from "../Components/FactSubList";
+import {
+    jobBoards,
+    professionalTraining,
+    housingAdvisory,
+    shelterListings,
+    studentAid,
+    postIncarEduPrograms,
+    otherEduAssistance,
+} from "../data/resourcesData";
+import FactGroup from "../Components/FactGroup";
+import { useDocumentTitle } from "../Components/customHooks/useDocumentTitle";
+
+const employmentFacts = [
+    {
+        id: "emp-fact-1",
+        src: time,
+        title: "Long periods of unemployment",
+        text: "Getting back into society may take longer with a conviction on record. ",
+    },
+    {
+        id: "emp-fact-2",
+        src: emails,
+        title: "Rejected applications",
+        text: "These are a common circumstance for people who have previously been convicted.",
```

Comment:
Is the grammar correct here? In my mind it should read, "This is a common circumstance..."

Reply:
Done! Changed it to "This is a common circumstance"